argocd/argocd/values.yaml

```diff
@@ -1,12 +1,14 @@
 ---
-# -- Values passed to the argo-cd chart
 argo-cd:
-  controller:
+  controller: 
     metrics:
       serviceMonitor:
+        # -- Specifies whether to enable Prometheus metrics of ArgoCD controller
```

Comment:
Again here, we want to keep the top-level value comment only, so we don't document the `argo-cd` chart.

We *could* link to the doc for this chart on top though, if such a doc exists.